timeline/plugins.py

```diff
@@ -114,3 +114,28 @@ class SiteAppPlugin(SiteAppPluginPoint):
 
     #: Required permission for displaying the app
     app_permission = 'timeline.view_site_timeline'
+
+
+class AdminSiteAppPlugin(SiteAppPluginPoint):
+    """Projectroles plugin for registering the app"""
+
+    #: Name (slug-safe, used in URLs)
+    name = 'admin_site'
+
+    #: Title (used in templates)
+    title = 'Admin Site Events'
+
+    #: App URLs (will be included in settings by djangoplugins)
+    urls = urlpatterns
+
+    #: Iconify icon
+    icon = 'mdi:clock-time-eight'
```

Comment:
I suggest choosing a different icon for this to differentiate it from the site-wide events view. `mdi:clock-time-eight` and `mdi:clock-time-eight-outline` are already in use elsewhere by the timeline app, so how about going with some other clock icon in Material Design Icons? You can browse them here: https://materialdesignicons.com/